main.js

```diff
@@ -122,20 +121,31 @@ function hideImage() {
   bellImage.classList.add('hidden');
```

Comment:
displayForm... are those backticks on line 112? Does it still work or should they be single quotes?

Reply:
It doesn't work with the single quotes. I think what is happening is it is referencing my currentMessage element which also uses backticks to interpolate... but that is honestly just a guess. It works with the backticks though, so thats a win!